README.md

```diff
@@ -1 +1,69 @@
-# powsybl-study-app
\ No newline at end of file
+# powsybl-study-app
```

Comment:
The default degenerated readme is not intended to be kept as it is. Either we don't have a readme or we write a dedicated one.